specs/gpad-gpi-2-0.md

```diff
@@ -196,16 +197,22 @@ Each annotation is on a separate line of tab separated values:
  5 | Reference ::= [ID] ('\|' ID)* | | 1 or greater | PMID:30695063 | Different IDs, e.g. PMID and MOD paper id, must correspond to the same publication or reference
  6 | Evidence_type ::= OBO_ID | Evidence and Conclusion Ontology | 1 | ECO:0000315 |  Mapping file in progress:  https://github.com/evidenceontology/evidenceontology/issues/249
  7 | With_or_From ::= [ID] ('\|' \| ‘,’ ID)* | | 0 or greater | WB:WBVar00000510 | Pipe-separated entries represent independent evidence; comma-separated entries represent grouped evidence, e.g. two of three genes in a triply mutant organism
- 8 | Interacting_taxon_ID ::= NCBITaxon:[Taxon_ID] | | 0 or greater | NCBITaxon:5476 |
+ 8 | Interacting_taxon_ID ::= ['NCBITaxon:'Taxon_ID] (',' 'NCBITaxon:'[Taxon_ID])* | | 0 or greater | NCBITaxon:5476 |
```

Comment:
It that `| |`? maybe `| '|'`?

Reply:
Ah you're right! Should be comma or pipe (https://github.com/geneontology/go-annotation/issues/3031#issuecomment-641351411). I'll update.

Reply:
@kltm I updated to match how it is done in With_or_From.